### DISCUSSION TITLE
(REQUEST) Stripped logs

### DISCUSSION THREAD

**Discussion by Cronosus** [Roadhog360/Et-Futurum-Requiem#517] [Mod Support Suggestion]
### Initial Questions

- [ ] I have searched this issue tracker and there is nothing similar already, and this is not on the [list of known issues](https://github.com/Roadhog360/Et-Futurum-Requiem/issues/146). Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I have verified that this issue occurs in a SUPPORTED environment, meaning I can reproduce this WITHOUT OptiFine, and without Bukkit+Forge server software like KCauldron, Thermos, Crucible, Mohist etc

### Mod Version

2.6.0

### Describe the Issue

Hello
Not issue, just have request, if it would be possible to add support for own stripped logs. 
like, i use customthings mod and added 8 variants of logs, together with their stripped variants. now i use chisel to swap, but it would be nice if i could simply add them into etfuturum config so it would use its function (by axe) to strip those logs.
example: <customthings:block6> strips into <customthings:block6:5>

thanks in advance

**Comment by Roadhog360** [comment]
Mod support is planned yeah. In your case though the content would have to be data-driven to work.
